src/templates/sections/categories/categories.css

```diff
@@ -11,12 +11,29 @@
 
 .category-tile {
   flex: 1; /* Allow tiles to grow and fill available space */
-  min-width: 900px; /* Set a minimum width to ensure proper size */
+  min-width: 300px; /* Set a minimum width to ensure proper size */
   height: 30vh;
   border-radius: 2vw;
   justify-content: center;
   align-items: center;
   transition: transform 0.2s, background-color 0.2s;
+  display: flex; /* Flexbox to center content */
+}
+
+.category-tile.men {
+  background-color: #2bd9af;
+}
+
+.category-tile.women {
+  background-color: #ff5e84;
+}
+
+.category-tile.men:hover {
+  background-color: #239b85;
+}
+
+.category-tile.women:hover {
+  background-color: #cc4b69;
 }
```

Comment:
@N-i-z We have used the same colors in `ProductCard` as well—so it's better to unify these background-color classes into a common css file! You can call it `theme.css` maybe... 

Define these colors in one file & re-use these classes when needed!